app/medInria/medRegistrationWorkspace.cpp

```diff
@@ -66,62 +47,62 @@ void medRegistrationWorkspace::setupViewContainerStack()
     //the stack has been instantiated in constructor
     if (!this->stackedViewContainers()->count())
     {
-        d->fixedContainer = this->stackedViewContainers()->addContainerInTab(tr("Compare"));
+        d->containers.push_back(this->stackedViewContainers()->addContainerInTab(tr("Compare")));
```

Comment:
I would avoid using push_back here as the order is important (fixed, moving, fuse), which means moving the code around will cause problems. Place the elements directly at their intended index (you'll need to explicitly set the vector size to 3 before).